### PR TITLE
fix: solve github parser bug

### PR DESCRIPTION
**Pull request by liqun98**
see: #75

### CODE REVIEW

File: github_poster/html_parser/github_parser.py [hunk 1]
```diff
@@ -4,11 +4,23 @@
...
+        self.recording = False
         self.rects = []
 
     def handle_starttag(self, tag, attrs):
         if tag == "rect":
             self.rects.append(dict(attrs))
+            self.recording = True
+
+    def handle_data(self, data):
+        if self.recording == True:
```
Comment (yihong0618) [inline github_poster/html_parser/github_parser.py:16]: if self.recording:

**Review by yihong0618 — APPROVED** [review]

**Comment by yihong0618** [comment]
LGTM.

use `black` to fix the lint would be better

**Comment by yihong0618** [comment]
lint error;
image

**Comment by yihong0618** [comment]
very helpful and neat `PR`
thanks a lot

**Comment by liqun98** [comment]
Sorry, I have not used black before, I'will try to fix lint in my local environment until the fix is done.

**Comment by yihong0618** [comment]
No worry.

**Comment by liqun98** [comment]
I use black to format this file, and it seems all things done. Please check it.

**Comment by yihong0618** [comment]
merge first, ci problems are mine.
Thanks a lot

**Pull request merged**